packages/grafana-data/src/transformations/fieldReducer.ts

```diff
@@ -343,6 +377,10 @@ export function doStandardCalcs(field: Field, ignoreNulls: boolean, nullAsZero:
         if (currentValue < calcs.logmin && currentValue > 0) {
           calcs.logmin = currentValue;
         }
+
+        let _oldMean = calcs.mean;
+        calcs.mean += (currentValue - _oldMean) / calcs.nonNullCount;
+        squareSum += (currentValue - _oldMean) * (currentValue - calcs.mean);
```

Comment:
can this overflow if you needed to sum 3000 datapoints where each value was in the 1e10 magnitude?